docs/DeveloperGuide.md

```diff
@@ -63,6 +64,19 @@ Our project follows the [Airbnb Javascript Style Guide](https://github.com/airbn
 
 Eslint and its accompaning modules can be installed through NPM, so do ensure that you got it [installed](https://www.npmjs.com/get-npm) if you are working on the dashboard.
 
+### Configuring Cypress for automated front-end testing
+We use [Cypress](https://www.cypress.io/) for automated end-to-end front-end testing. <br/>
+
+#### To write tests
+1. Create a new test file in `./frontend/cypress/tests`
+1. At project root start *Cypress Test Runner* by running `./gradlew cypress`
+1. Click on the newly created test file under **Integration Testing**
+> Read [Cypress's Documentation](https://docs.cypress.io/api/commands/document.html#Syntax) to familiarize yourself with its syntax and [Cypress's debugging guide](https://docs.cypress.io/guides/guides/debugging.html#Log-Cypress-events) to tackle problems with your tests.
+
+#### To run all tests locally
+1.  At project root, run `./gradlew uat`
```

Comment:
Let's use `gradlew uat` without the `./`, to follow the same format used throughout the DG. Same for above `gradlew cypress`.